spec/factories/payments.rb

```diff
@@ -0,0 +1,27 @@
+FactoryBot.define do
+  factory :payment do
```

Comment:
since you define factory, will be good to add test, which checks if factory valid